README.md

```diff
@@ -1,30 +1,75 @@
-# Data Loading
-
-The {dv.loader} package provides a simple interface for loading data from a network file storage folder or 
-locally. It is designed to be used with `.RDS` and `.sas7bdat` file formats.
-The package provides a simple function, `load_data()`, which loads R and SAS data files into memory. 
-Loading data from SQL databases is not yet supported. The function returns a list named by the file names passed, 
-and containing data frames, along with metadata for that table. By default, the function will look for files in a
-sub-directory `sub_dir` of the base path defined by a environment variable "RXD_DATA". You can check if the base path
-is set by running `Sys.getenv("RXD_DATA")`. A single file or multiple files can be loaded at once. 
-To make the loading process faster for large datasets, it is suggested that '.sas7bdat' files are converted to 
-'.RDS' files. The function will prefer '.RDS' files over '.sas7bdat' files by default.
+# dv.loader
+
+The `dv.loader` package is designed to simplify the data loading process for creating modular Shiny applications within the DaVinci framework.
+
+Key features of `dv.loader` include:
+
+1. Capability to import multiple files from a data directory.
+
+2. Compatibility with both `.rds` and `.sas7bdat` file types.
+
+3. Smooth integration with other packages in the DaVinci ecosystem.
+
 
 ## Installation
 
+You can install the `dv.loader` package from GitHub using the `remotes` package:
+
 ```r
+# Install the remotes package if not already installed
 if (!require("remotes")) install.packages("remotes")
+
+# Install the dv.loader package from GitHub
 remotes::install_github("Boehringer-Ingelheim/dv.loader")
 ```
 
-## Basic usage
+## Examples
+
+The main function is `dv.loader::load_data()`, which loads data files from sub-directories of a network file system (NFS) or the working directory.
```

Comment:
As pointed out elsewhere, every reference to a "network file system" should be avoided. `dv.loader` doesn't know or care about networking. Notice that the old documentation avoided this issue by expanding it to `network file storage folder`.

Reply:
The documentation now uses "base path" terminology as suggested.